components/05-page-examples/_intro-content-examples.twig

```diff
@@ -1,25 +1,41 @@
 {% if intro_content == 'image' %}
   {% include "@molecules/image/content-image.twig" %}
-{% elseif intro_content =='image--highlight' %}
+{% elseif intro_content == 'image--highlight' %}
```

Comment:
I wish Twig would implement a switch case statement. Other CMS's implemented this functionality but it would be a perfect fit for this file. https://www.branchcms.com/learn/docs/developer/twig/tags/item/switch

Reply:
I agree